zeppelin-web/src/app/notebook/paragraph/paragraph-control.html

```diff
@@ -54,20 +55,6 @@
           ng-click="cancelParagraph(paragraph)"
           ng-class="{'item-disable' : isNoteRunning}"
           ng-show="paragraph.status=='RUNNING' || paragraph.status=='PENDING'"></span>
-    <span ng-show="paragraph.runtimeInfos.jobUrl.length == 1">
-      <a href="{{paragraph.runtimeInfos.jobUrl[0]}}" target="_blank"><span class="fa fa-tasks"></span> Spark job </a>
-    </span>
-    <span class="dropdown" ng-show="paragraph.runtimeInfos.jobUrl.length > 1">
-      <span class="fa fa-tasks" style="cursor:pointer;color:#3071A9" tooltip-placement="top" uib-tooltip="Run this paragraph (Shift+Enter)"
-          data-toggle="dropdown"
-          type="button">  Spark Jobs
-      </span>
-      <ul class="dropdown-menu" role="menu" style="width:200px;z-index:1002">
-         <li ng-class="{'option-disabled': !noteOperationsAllowed()}" ng-repeat="url in paragraph.runtimeInfos.jobUrl">
-           <a href="{{url}}" target="_blank"><span class="fa fa-external-link-square"></span> Jobs #{{$index}}</a>
-         </li>
-      </ul>
```

Comment:
how is this getting shown now?

Reply:
ok, I thought the Spark UI link is there before. has that been broken?

Reply:
The valid spark ui code is here. 
`https://github.com/apache/zeppelin/blob/master/zeppelin-web/src/app/notebook/paragraph/paragraph-control.html#L24`

The code here is repetitive, And it won't show up.
`https://github.com/apache/zeppelin/blob/master/zeppelin-web/src/app/notebook/paragraph/paragraph-control.html#L60`

So can delete `paragraph-control.html#L60` without affecting the display of the spark UI.